src/main/kotlin/org/openmbee/flexo/mms/routes/ldp/OrgRead.kt

```diff
@@ -0,0 +1,147 @@
+package org.openmbee.flexo.mms.routes.ldp
+
+import io.ktor.http.*
+import io.ktor.server.response.*
+import org.openmbee.flexo.mms.*
+import org.openmbee.flexo.mms.server.LdpDcLayer1Context
+import org.openmbee.flexo.mms.server.LdpGetResponse
+import org.openmbee.flexo.mms.server.LdpHeadResponse
+import org.openmbee.flexo.mms.routes.SPARQL_VAR_NAME_ORG
+
+private const val SPARQL_VAR_NAME_CONTEXT = "_context"
+
+// reusable basic graph pattern for matching org(s)
+private val SPARQL_BGP_ORG = """
+    graph m-graph:Cluster {
+        ?$SPARQL_VAR_NAME_ORG a mms:Org ;
+            mms:etag ?__mms_etag ;
+            ?org_p ?org_o .
+        
+        optional {
+            ?thing mms:org ?$SPARQL_VAR_NAME_ORG ;
+                ?thing_p ?thing_o .
+        }
+    }
+    
+    ${permittedActionSparqlBgp(Permission.READ_ORG, Scope.CLUSTER)}
+"""
+
+// select ETag(s) of existing org(s)
+private val SPARQL_SELECT_ORG_ETAGS = """
+    select distinct ?__mms_etag {
+        $SPARQL_BGP_ORG
+    } order by asc(?__mms_etag)
+"""
+
+// construct graph of all relevant org metadata
+private val SPARQL_CONSTRUCT_ORG = """
+    construct {
+        ?$SPARQL_VAR_NAME_ORG ?org_p ?org_o ;
+            mms:etag ?__mms_etag .
+        
+        ?thing ?thing_p ?thing_o .
```

Comment:
shouldn't the org read just return the org information and ldp:contains to the repo iris (since there's already repo read) - can be a separate issue to clean up what all the ldp read returns (also see comment on repo read returning the entire metadata graph)

Reply:
created issue #159